src/auditwheel/main_addtag.py

```diff
@@ -30,7 +30,10 @@ def execute(args, p):
     try:
         wheel_abi = analyze_wheel_abi(args.WHEEL_FILE)
     except NonPlatformWheel:
-        logger.info("This does not look like a platform wheel")
+        logger.info(
+            "This does not look like a platform wheel"
+            ", no compiled extension file found in the wheel archive"
```

Comment:
auditwheel does not require the presence of a compiled extension. It requires the presence of at least 1 ELF file: either an executable or a shared library. A compiled extension is just a specific kind of shared library.

Reply:
I guess a compiled extension is the most common usage kind for cibuildwheel, but I agree that we should be more general.

Does the following work?

> , no executable or shared library found in the wheel archive